project-base/tests/FrontendApiBundle/Test/GraphQlTestCase.php

```diff
@@ -205,4 +235,124 @@ protected function assertResponseContainsArrayOfExtensionValidationErrors(array
+    }
+
+    /**
+     * @param string $priceWithoutVat
+     * @param string $vatRate
+     * @param int $quantity
+     * @return \Shopsys\FrameworkBundle\Model\Pricing\Price
+     */
+    private function getConvertedPriceToDomainDefaultCurrency(
+        string $priceWithoutVat,
+        string $vatRate,
+        int $quantity = 1
+    ): Price {
+        $domainId = $this->domain->getId();
+        $vat = $this->getVatByVatRate($vatRate);
+        $currency = $this->currencyFacade->getDomainDefaultCurrencyByDomainId($domainId);
+
+        $basePrice = $this->basePriceCalculation->calculateBasePriceRoundedByCurrency(
+            $this->priceConverter->convertPriceWithoutVatToPriceInDomainDefaultCurrency(
+                Money::create($priceWithoutVat),
+                $domainId
+            ),
+            PricingSetting::INPUT_PRICE_TYPE_WITHOUT_VAT,
+            $vat,
+            $currency
+        );
+
+        return $this->basePriceCalculation->calculateBasePriceRoundedByCurrency(
+            $basePrice->getPriceWithVat()->multiply($quantity),
+            PricingSetting::INPUT_PRICE_TYPE_WITH_VAT,
+            $vat,
+            $currency
+        );
+    }
+
+    /**
+     * @param string $priceWithoutVat
+     * @param string $vatRate
+     * @param int $quantity
+     * @return array
+     */
+    protected function getSerializedPriceConvertedToDomainDefaultCurrency(
+        string $priceWithoutVat,
+        string $vatRate,
+        int $quantity = 1
+    ): array {
+        $price = $this->getConvertedPriceToDomainDefaultCurrency($priceWithoutVat, $vatRate, $quantity);
+        return [
+            'priceWithVat' => MoneyFormatterHelper::formatWithMaxFractionDigits($price->getPriceWithVat()),
+            'priceWithoutVat' => MoneyFormatterHelper::formatWithMaxFractionDigits($price->getPriceWithoutVat()),
+            'vatAmount' => MoneyFormatterHelper::formatWithMaxFractionDigits($price->getVatAmount()),
+        ];
+    }
+
+    /**
+     * @param string $priceWithoutVat
+     * @param string $vatRate
+     * @param int $quantity
+     * @return string
+     */
+    protected function getMutationPriceConvertedToDomainDefaultCurrency(
+        string $priceWithoutVat,
+        string $vatRate,
+        int $quantity = 1
+    ): string {
+        $price = $this->getConvertedPriceToDomainDefaultCurrency($priceWithoutVat, $vatRate, $quantity);
+        return '{
+            priceWithVat: "' . MoneyFormatterHelper::formatWithMaxFractionDigits($price->getPriceWithVat()) . '",
+            priceWithoutVat: "' . MoneyFormatterHelper::formatWithMaxFractionDigits($price->getPriceWithoutVat()) . '",
+            vatAmount: "' . MoneyFormatterHelper::formatWithMaxFractionDigits($price->getVatAmount()) . '"
+        }';
+    }
+
+    /**
+     * @param array $expectedOrderItems
+     * @return array
+     */
+    protected function getSerializedOrderTotalPriceByExpectedOrderItems(array $expectedOrderItems): array
```

Comment:
Actually I was thinking about that too. But it could not be used for testing getting orders as authenticated customer.

I think we should create a helper with these kind of methods. What do you say?

Reply:
Or we could create it static in `AbstractOrderTestCase`. In that case usages elsewhere would not be a problem any more.

Reply:
moved in 38d992c6166c91b2fa80d4823c41b0efb9938e37